code/game/gamemodes/scores.dm

```diff
@@ -0,0 +1,393 @@
+#define MAX_FACTION_SCORE 1000
```

Comment:
Faction score should be uncapped if it's just a number so we can easily spot exploits(like the guild infinite money with surplus ammo), but if it's going to get tied to some other important system (like irska coins) it should be capped.

Reply:
This is not a cap, it is simply a maximum reference value. If the score is less than half this value, it means that a good job has not been done in the round.